grails-app/controllers/au/org/ala/biocache/hubs/OccurrenceController.groovy

```diff
@@ -62,9 +63,10 @@ class OccurrenceController {
     def list(SpatialSearchRequestParams requestParams) {
         def start = System.currentTimeMillis()
 
-        def activeProfile = time("active profile") { qualityService.activeProfile(requestParams.qualityProfile) }
-
-        normaliseRequestParams(requestParams)
+        // get the user preference settings
+        def userPref = userService.getUserPref(authService?.getUserId(), request)
```

Comment:
This code is not surrounded by check if `dataquality.enabled == true`. Should it be? Checking if you have run it with that config set to false?

Reply:
I'll check, basically anywhere dq related, check if dq is enabled.